src/cvf.py

```diff
@@ -53,12 +53,40 @@ def get_paper_page_urls(conference: str, year: int) -> list[str]:
     """
     cvf_root_url: Final[str] = "https://openaccess.thecvf.com"
     conference_name: Final[str] = validate_conference(conference, year)
-    cvf_all_paper_url: Final = cvf_root_url + f"/{conference_name}?day=all"
 
-    html: Final = requests.get(cvf_all_paper_url).text
-    bs: Final = BeautifulSoup(html, "html.parser")
-    parsed_tags = bs.select(".ptitle > a")
-    return [cvf_root_url + parsed_tag.get("href") for parsed_tag in parsed_tags]
+    # 発表日ごとの表示しか出来ない場合
+    if (conference == "cvpr" and year == 2020) or (
+        conference == "iccv" and year == 2019
+    ):
```

Comment:
```suggestion
    if int(year) <= 2000:
```
2000年より前はCVPRもICCVも発表日ごとの表示しか出来ないので、これで良いんじゃないかな？